docs/reference/objects/product/index.md

```diff
@@ -83,6 +83,13 @@ string
 
 The duration of the product including duration unit e.g. `5 nights`.
 
+## `product.experience_dates`
+{: .d-inline-block }
+[ExperienceDate]({% link docs/reference/objects/product/experience_date.md %})
+{: .label .fs-1 }
+
+An array of the product's [experience dates]({% link docs/reference/objects/product/experience_date.md %})
```

Comment:
Nit: can we add a `.` at the end?

Reply:
Should we reference that this is only works when the `type` is experience? Or something like that?